merino/utils/gcs/async_gcs_client.py

```
@@ -1,18 +1,16 @@
 """Async client that usese community maintained gcloud-aio library to interact with Google Cloud Storage"""
 
 import logging
-import orjson
-from orjson import JSONDecodeError
-
 from gcloud.aio.storage import Storage
-from merino.providers.manifest.backends.protocol import ManifestData
 
 
 logger = logging.getLogger(__name__)
 
 
 class AsyncGcsClient:
-    """Class that provides wrapper functions around gcloud-aio-storage functions. More functionality to be added."""
+    """Class that provides wrapper functions around gcloud-aio-storage functions.
```

Do we still need this wrapper? Looks like all it does is wrap the `download` call with some additional logging. Do we also use this to facilitate testing (e.g. dependency ingestion)?

---

Also noticed that there was exception handling in `__init__()`. We don't normally handle exceptions in the constructor. If it fails in the rare cases, just let it fail. Those errors will be captured and reported by Sentry for further investigation. You can reference `utils/http_client.py` for that.